### PR TITLE
Feature/aptos sign fallback

### PR DESCRIPTION
**Pull request by mordochi**
## Summary

Support SDK fallback of signing messages on Aptos
## Related Links

**Spec**: https://www.notion.so/portto/Aptos-SDK-fallback-725ad5232dae42efa5a2250c04922448

**Asana**:  https://app.asana.com/0/1201812548509877/1204004296893665/f

**Mockup**:  

## Checklist

- [x] Pasted Asana or Mockup link.
- [x] Tagged labels.

## Prerequisite/Related Pull Requests

https://github.com/portto/wallet-web-backend/pull/35

### CODE REVIEW

File: src/features/Aptos/Signing/components/Connecting.tsx [hunk 1]
```diff
@@ -18,53 +19,75 @@ const Connecting = () => {
...
-                application,
-                chainId,
+    try {
+      if (signatureId) {
+        // get user type (custodial or not) and get the details of the signing message
+        Promise.all([
+          getUserInfo(),
+          getSignatureDetails({ blockchain, signatureId }),
+          getAccountAsset({ blockchain, force: true }),
+        ]).then(([{ type }, signatureDetails, { wallet_address }]) => {
```
Comment (yanglin5689446) [inline src/features/Aptos/Signing/components/Connecting.tsx:29]: prefer to make `wallet_address` reassigned to `walletAddress`:
```
{ wallet_address: walletAddress }
```
to make the style consistent

File: src/features/Aptos/Transaction/components/Connecting.tsx [hunk 2]
```diff
@@ -17,68 +17,68 @@ const Connecting = () => {
...
-        });
-    }
+      const assets = allAssets.filter(
+        (asset: AccountAsset) => asset.blockchain === blockchain
+      );
 
-    const assets = allAssets.filter(
-      (asset: AccountAsset) => asset.blockchain === blockchain
-    );
+      const [{ value: balance = 0, wallet_address }] = assets.filter(
```
Comment (yanglin5689446) [inline src/features/Aptos/Transaction/components/Connecting.tsx:38]: also here

**Review by yanglin5689446 — APPROVED** [review]

File: src/features/Aptos/Signing/components/Connecting.tsx (hunk 1 above)
Comment (mordochi) [reply src/features/Aptos/Signing/components/Connecting.tsx]: 6aa12a50a8236770d34b0fa9d15fa3fbe2c8f00b

File: src/features/Aptos/Transaction/components/Connecting.tsx (hunk 2 above)
Comment (mordochi) [reply src/features/Aptos/Transaction/components/Connecting.tsx]: 6aa12a50a8236770d34b0fa9d15fa3fbe2c8f00b

**Pull request merged**